frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -228,35 +221,38 @@ export default class Select extends Component {
             </SelectButton>
           )
         }
-        onClose={composeEventHandlers(onClose, this.handleClose)}
+        popoverContent={({ closePopover, maxHeight }) => (
```

Comment:
use `popoverContent` as this will wrap the content within a trigger button (now render as `<a>` tag to make it compatible with old E2E tests). I tried with `renderPopoverContent` but seems like it was intended for rendering the trigger element also, so doesn't automatically wrap the content within an `<a>` with specified `triggerClasses`

TippyPopover implemented the trigger element using `<button>` tag and it broke a lot of tests. e.g.
- https://app.circleci.com/pipelines/github/metabase/metabase/35669/workflows/6aa3d00b-6d8a-4802-8f3a-2501cbe765e4
- https://app.circleci.com/pipelines/github/metabase/metabase/35681/workflows/e164c5fe-cf45-4107-811f-f0e4fc4d40bc
- https://app.circleci.com/pipelines/github/metabase/metabase/35679/workflows/ab132235-bd6d-45ed-897e-0e13b70cea87